src/app/pages/BuySovPage/components/BuyForm/index.tsx

```diff
@@ -115,18 +115,25 @@ export function BuyForm() {
               onChange={value => setAmount(value)}
               placeholder="0.0000"
               rightElement={<AssetRenderer asset={Asset.RBTC} />}
+              dataAttribute="buySovPage-enterAmount-field"
             />
             <Slippage>
-              <AvailableBalance asset={Asset.RBTC} />
+              <AvailableBalance
+                asset={Asset.RBTC}
+                dataAttribute="buySovPage-availableBalance-field"
+              />
             </Slippage>
-            <AmountButton onChange={changeAmount} />
+            <AmountButton
+              onChange={changeAmount}
+              dataAttribute="buySovPage-amount-"
```

Comment:
Adding the trailing hyphen in the amount button would be cleaner.

Reply:
please use whatever u think is appropriate.

Reply:
> please use whatever u think is appropriate.

See https://github.com/DistributedCollective/Sovryn-frontend/pull/1715#discussion_r713820584